server/base/src/main/java/org/apache/accumulo/server/compaction/FileCompactor.java

```diff
@@ -212,9 +216,21 @@ public CompactionStats call() throws IOException, CompactionCanceledException {
     try {
       FileOperations fileFactory = FileOperations.getInstance();
       FileSystem ns = this.fs.getFileSystemByPath(outputFile.getPath());
-      mfw = fileFactory.newWriterBuilder()
+
+      boolean dropCacheBehindMajcOutput = false;
+      if (!RootTable.ID.equals(this.extent.tableId())
+          && !MetadataTable.ID.equals(this.extent.tableId())
+          && CompactionConfigurer.dropCacheBehindMajcOutput(acuTableConf)) {
+        dropCacheBehindMajcOutput = true;
+      }
```

Comment:
This is more concise.
```suggestion
      boolean dropCacheBehindMajcOutput = !RootTable.ID.equals(this.extent.tableId())
          && !MetadataTable.ID.equals(this.extent.tableId())
          && CompactionConfigurer.dropCacheBehindMajcOutput(acuTableConf);
```

Reply:
OBE, logic updated from changes in dba224e. Updated conditional with suggested change.